sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,408 @@
+
+            ### RMA provisioning
+            - Instantiate the SW CSRNG, and configure it to generate data.
+            - Configure RMA_TOKEN in the OTP partition with data extracted from the CSRNG.
+            - Encrypt RMA_TOKEN with asymmetric algorithm (e.g RSA-3072). The encrypted blob will
+              be sent to the host in the result status.
+            - Uninstantiate the sw CSRNG.
+
+            ### Device Identity
+
+            The OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN OTP must be configured so that the
+            attestation measurement can be provided in the manifest of the test program. This
+            way the provisioning program is able to generate the key manager derivation matching
+            the production ROM_EXT configuration.
+
+            TODO: Left details vague on purpose until we get the attestation flow reviewed.
+            - Advance the keymgr to CREATOR_ROOT_KEY state.
+            - Derive CREATOR attestation key pair.
+            - Calculate SHA2 hash of the public key with the device secret provisionined in
+              the manuf_cp_device_info_flash_wr test. The device secret is known by the
+              provisioning appliance.
+            - Disable the key manager.
+            - Configure OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN OTP to switch to attestation mode
+              associated with the device SKU.
+
+            ### Result status
+            - Send response to host with test result serialized in JSON format. The result must
+              include the encrypted RMA_UNLOCK_TOKEN, the DEVICE_ID, the public attestation key
+              and its hash.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_load_transport_image
+      desc: '''Load transport stack into flash.
+
+            Ensure the ROM is able to bootstrap the transport image. The transport image supports
+            functionality such as full device attestation, and ownership transfer, which are
+            required to enable device integration into a system.
+
+            - Pre-load the OTP with an image of a personalized device. See
+              manuf_ft_provision_rma_token_and_personalization for more details.
+            - Bootstrap transport image.
+            - Reset device by toggling POR pin.
+            - Check device firmware version.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_load_certificates
+      desc: '''Load endorsed certificates.
+
+            Load device certificate endorsing the creator public key. The certificate is signed
+            by the silicon creator.
+
+            TODO: This assumes the ROM_EXT supports a mechanism to update the certificate.
+            TODO: Should the transport image contain silicon creator pubilc keys to be able to
+            verify the certificate before install?
```

Comment:
What would happen if an invalid certificate was installed? As long as this operation can be retried, I'd suggest offloading this to the host. Since certificate install can fail _after_ on-device verification, this seems more like a convenience.